src/validator.ts

```diff
@@ -9,19 +9,26 @@ let defaultMessage = 'The value must be greater than or equal to {{minValue}}';
 export const setErrorMessage = (message: string) => (defaultMessage = message);
 
 interface CustomValidatorArgs {
+  strictTypes?: boolean;
   minValue: number;
   inclusive?: boolean;
 }
 
-const defaultCustomArgs: CustomValidatorArgs = {
+let defaultCustomArgs: CustomValidatorArgs = {
+  strictTypes: false,
   minValue: 0,
   inclusive: true,
 };
+export const setCustomArgs = (customArgs: CustomValidatorArgs) =>
```

Comment:
It means that we have to provide the `minvalue` when we only want to set `strictTypes = true` gobally. Maybe we should allow provide only strictTypes in this method (I don't know if `inclusive` flag too)

Reply:
I set the parameter as partial. Now we can partially override the default object.